Managers/PhaseManager.gd

```diff
@@ -14,7 +14,8 @@ var current_phase: GlobalEnums.Phase = GlobalEnums.Phase.NONE
 
 
 func _ready() -> void:
-	initialize_game()
+	# initialize_game()
```

Comment:
Is there a reason this is commented out?

Reply:
The phase manager is a singleton, meaning it's loaded first
It was fine to use `initialize_game` (which should probably be renamed to `init_combat` instead)
This is now commented out, since we init the combat when entering the scene (which is done via the scene switcher, and not at game start)